### PR TITLE
Hugo overhaul

### PR DESCRIPTION
**Pull request by Emersont1**

### CODE REVIEW

File: data/committee.yml [hunk 1]
```diff
@@ -2,32 +2,7 @@ members:
...
-      instagram: https://www.instagram.com/jtav00/
-    - identifier: tin-dog
-      title: Tin Dog
-      name: Zac Garby
-      image: n/a
-      website: https://zacgarby.co.uk
-      twitter: https://twitter.com/zacgarby
-      email: me@zacgarby.co.uk
\ No newline at end of file
+      email: pmy@pt3.uk
```
[the remark this answers is not in the document]
Comment (Fraetor) [reply data/committee.yml]: Here is an example newsletter markdown file. There is a tiny amount of HTML as well for the buttons.

[newsletter-2021-04-28.md](https://github.com/HackSocNotts/hacksocnotts.github.io/files/6735845/newsletter-2021-04-28.md)

**Comment by Emersont1** [comment]
https://github.com/HackSocNotts/hacksocnotts.github.io/pull/176#discussion_r660898158

Just pasting it in thread so @zac-garby can find it

File: data/committee.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (zac-garby) [reply data/committee.yml]: Hugo doesn't seem to like the HTML very much, I might be able to do some setting to allow it through or else I'll probably have to add a preprocessing step to convert them into markdown.

[the remark this answers is not in the document]
Comment (Fraetor) [reply data/committee.yml]: The other option I though of was to take the HTML document produced by the newsletter make and embed that in an iframe.

[the remark this answers is not in the document]
Comment (zac-garby) [reply data/committee.yml]: Oh if I could get the newsletter HTMLs instead, Hugo can use that directly. It just doesn't like mixing md and html

**Comment by Fraetor** [comment]
Here's a HTML newsletter. See if that works better, and then I can regenerate them all for you.
[newsletter-2021-06-29.zip](https://github.com/HackSocNotts/hacksocnotts.github.io/files/6742671/newsletter-2021-06-29.zip)
(Zipped becasue GitHub doesn't like HTML files.)

**Comment by zac-garby** [comment]
> Here's a HTML newsletter. See if that works better, and then I can regenerate them all for you.
> [newsletter-2021-06-29.zip](https://github.com/HackSocNotts/hacksocnotts.github.io/files/6742671/newsletter-2021-06-29.zip)
> (Zipped becasue GitHub doesn't like HTML files.)

That works perfectly. Image scaling works better than it did with the .md as well.

**Comment by Emersont1** [comment]
I think we just need to add photos now

**Comment by zac-garby** [comment]
do we have photos for everyone?

**Comment by zac-garby** [comment]
also i want to get the backlog of newsletters uploaded, and also i need to get the constitution links and stuff like that working because i don't think i linked them yet

**Review by zac-garby — APPROVED** [review]

**Comment by Fraetor** [comment]
@zac-garby Should we upload photos to this issue?
Also, shall I get you all of the newsletters' HTML?

**Comment by Emersont1** [comment]
I'll make a new PR for scaled down versions of the images

**Comment by zac-garby** [comment]
i'll actually make a new issue for the photo uploads, just to organise things a bit. and yeah if you could, thanks

**Comment by Emersont1** [comment]
having a fairly full backlog of newsletters would be great

**Pull request merged**